dali/base/dacsds.cpp

```diff
@@ -1776,6 +1778,8 @@ IRemoteConnection *CClientSDSManager::connect(const char *xpath, SessionId id, u
             mb.read(connId);
             conn = new CRemoteConnection(*this, connId, xpath, id, mode & ~RTM_CREATE_MASK, timeout);
             assertex(conn);
+            if (queryProperties().getPropBool("Client/@LogConnection"))
```

Comment:
minor: a bit of a shame to be checking a IPropertyTree per connection.
I'm surprised there are others actually (in the constructor), e.g. serverIterAvailable - which is static. It should really setup once in a member variable in the manager (CClientSDSManager) and check that.

For this one, which could change, setConfigOpt() could spot and set member variable.

Probably not worth changing for this PR, another PR could move this one and the pre-existing ones (serverIterAvailable, serverIter, useAppendOpt, serverGetIdsAvailable) at the same time.